compose-templates/docker-compose-nginx-ssl.yml

```diff
@@ -12,6 +12,7 @@ services:
     secrets:
       - cert.pem
       - key.pem
+      # Generate this file with the command: "openssl dhparam -out dhparam.pem 4096"  
```

Comment:
I would add this to the deployment guide explicitly as well

Reply:
Yes, please!